doc/how_to_guides/usage.md

```diff
@@ -33,64 +34,83 @@ Runtime options:
 ```bash
 bundle exec rails generate onesie:task MyTask      # creates a normal priority task
 bundle exec rails generate onesie:task MyTask high # creates a high priority task
+
 ```
 
 ### Rake
+The following commands should occur within a v2 shell
+(`dcc shell benchprep-v2`). For more information on setting up
+containers, please consult our
+[container setup guide](../infrastructure/how_to_guides/dev_containers/setup_guide.md).
+
+You may notice that some of these examples include an optional
+template name as their last argument. These meta-templates are
+created by developers here at BenchPrep to further facilitate the
+creation of common types of onesies. For more information on finding,
+using, and contributing onesie templates, please refer to our
+[templates](/templates.md) documentation.
 
 #### API
 ```bash
-rake onesie:new[name,priority,template]         # Generates a new Onesie Task
+rake onesie:new[name,priority,template]         # Generates a new onesie task
+
 ```
 
 #### Examples
 
 ```bash
 bundle exec rake onesie:new['MyTask']        # creates a normal priority task
 bundle exec rake onesie:new['MyTask','high'] # creates a high priority task
-bundle exec rake onesie:new['MyTask','high','SampleTemplate'] # creates a high priority task using 'SampleTemplate'
+bundle exec rake onesie:new['MyTask',,'SampleTemplate'] # creates a normal priority task using 'SampleTemplate'
```

Comment:
I thought it might be helpful to demonstrate leaving the priority blank (rather than repeating the priority specification from the previous example). I think this will be the most common use.